src/marten/cli/manage/command/collect_assets.cr

```diff
@@ -42,16 +77,36 @@ module Marten
             if collected_count == 0
               print("No assets to collect...")
             end
+
+            if fingerprint? && collected_count > 0
+              FileUtils.mkdir_p(Path[@manifest_path].dirname) # Ensure path exists
+
+              print("  › Creating #{style(@manifest_path, mode: :dim)}...", ending: "")
```

Comment:
Ideally, I think it would be better to show a relative manifest path here in order to ensure. that the command output is not too verbose. What about:

```suggestion
              relative_manifest_path = Path[@manifest_path].relative_to(Marten::Apps::Config.compilation_root_path)
              print("  › Creating #{style(relative_manifest_path, mode: :dim)}...", ending: "")

```

That way we will show the path of the manifest relative to the Marten root folder.